lcc_lokavaluto_app_connection/tests/test_res_alt_currency.py

```diff
@@ -0,0 +1,23 @@
+from odoo.addons.component.tests.common import TransactionComponentCase
+
+
+class TestResAltCurrency(TransactionComponentCase):
+    def setUp(self):
+        super().setUp()
+
+        self.ResUsers = self.env["res.users"]
+        self.ResPartner = self.env["res.partner"]
+        self.ResPartnerBackend = self.env["res.partner.backend"]
+        self.ResAltCurrency = self.env["res.alt.currency"]
+
+        self.currency_A_unit_product = self.env.ref(
+            "lcc_lokavaluto_app_connection.product_product_numeric_lcc"
+        ).sudo()
+        self.currency_A = self.ResAltCurrency.create(
```

Comment:
This should be a test not the setUp. Using `.create()` on the model you create is a test. (and otherwise, you effectively shipped your mode without tests...).